src/puppetlabs/trapperkeeper/services/status/status_service.clj

```diff
@@ -33,6 +33,11 @@
       (add-ring-handler this handler))
     context)
 
+  (stop [this context]
+    (status-core/dissoc-status-context! (:status-fns context)
+                                        status-core/status-service-name)
+    context)
```

Comment:
1. i'm totally fine with that, was just trying to do the simplest change first
2. `(tka/stop app) (tka/start app)`